lib/gooddata/lcm/actions/provision_clients.rb

```diff
@@ -33,16 +33,16 @@ class ProvisionClients < BaseAction
 
       class << self
         def call(params)
+          synchronize_projects = []
+          data_product = params.data_product
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)